src/src/com/microsoft/aad/adal/HttpWebRequest.java

```diff
@@ -182,8 +182,10 @@ public HttpWebResponse send() {
                 Logger.v(TAG, "Response is received");
                 response.setBody(responseBody);
                 response.setResponseHeaders(mConnection.getHeaderFields());
-            } catch (Exception e) {
-                Logger.e(TAG, "Exception:" + e.getMessage(), " Method:" + mRequestMethod,
+            } catch (InterruptedException ignore) {
```

Comment:
Ok, I'll rename it to e, but this Exception needs to be ignores. it only happens if Thread.sleep got interruption and we don't do anything about it and this method is only for debug purposes there (simulate slow networks)